src-ts/lib/work-provider/work-functions/work-factory/work.factory.ts

```diff
@@ -289,3 +289,75 @@ function getTypeCategory(type: WorkType): WorkTypeCategory {
             return WorkTypeCategory.unknown
     }
 }
+
+export function mapFormData(type: string, formData: any): any {
+    switch (type) {
+        case (WorkType.problem):
+            return buildFormDataProblem(formData)
+        case (WorkType.data):
+            return buildFormDataData(formData)
+        case (WorkType.findData):
+            return buildFormDataFindData(formData)
+        case (WorkType.design):
+            return buildFormDataDesign(formData)
+        default:
+            return formData
+    }
+}
+
+function buildFormDataProblem(formData: any): any {
+    return {
+        projectTitle: formData.projectTitle,
+        // tslint:disable-next-line: object-literal-sort-keys
+        goal: { title: 'What\'s Your Goal?', value: formData.goals?.value },
+        data: {
+            title: 'What Data Do You Have?', value: [
+                formData.sampleData?.value,
+                formData.assetsDescription?.value,
+            ],
+        },
+    }
+}
+
+function buildFormDataData(formData: any): any {
+    return {
+        projectTitle: formData.projectTitle,
+        // tslint:disable-next-line: object-literal-sort-keys
+        data: { title: 'Share Your Data (Optional)', value: formData.assetsUrl?.value },
+        goal: { title: 'What Would You Like To Learn?', value: formData.goals?.value },
+    }
+}
+
+function buildFormDataFindData(formData: any): any {
+    return {
+        projectTitle: formData.projectTitle,
+        // tslint:disable-next-line: object-literal-sort-keys
+        data: formData.analysis,
+        primaryDataChallenge: {
+            title: formData.primaryDataChallenge?.title,
+            value: formData.primaryDataChallenge?.value === 3 ?
```

Comment:
What significance is `3`? Let's pull it out into a properly named constant. Also, I think the standard formatting for a ternary operator with long arguments is to spread it over three separate lines, like this:

```
value: formData.primaryDataChallenge?.value === 3 ?
    formData.primaryDataChallengeOther.value : 
    formData.primaryDataChallenge?.option
```